examples/next/contexts/WalletSelectorContext.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useCallback, useContext, useEffect, useState } from "react";
+import { map, distinctUntilChanged } from "rxjs";
+import { setupWalletSelector } from "@near-wallet-selector/core";
+import type { WalletSelector, AccountState } from "@near-wallet-selector/core";
+import { setupModal } from "@near-wallet-selector/modal-ui";
+import type { WalletSelectorModal } from "@near-wallet-selector/modal-ui";
+import { setupNearWallet } from "@near-wallet-selector/near-wallet";
+import { setupMyNearWallet } from "@near-wallet-selector/my-near-wallet";
+import { setupSender } from "@near-wallet-selector/sender";
+import { setupMathWallet } from "@near-wallet-selector/math-wallet";
+import { setupNightly } from "@near-wallet-selector/nightly";
+import { setupLedger } from "@near-wallet-selector/ledger";
+import { setupWalletConnect } from "@near-wallet-selector/wallet-connect";
+import { setupNightlyConnect } from "@near-wallet-selector/nightly-connect";
+import { CONTRACT_ID } from "../constants";
+
+declare global {
+  interface Window {
+    selector: WalletSelector;
+    modal: WalletSelectorModal;
+  }
+}
+
+interface WalletSelectorContextValue {
+  selector: WalletSelector;
+  modal: WalletSelectorModal;
+  accounts: Array<AccountState>;
+  accountId: string | null;
+}
+
+const WalletSelectorContext =
+  React.createContext<WalletSelectorContextValue | null>(null);
+
+export const WalletSelectorContextProvider: React.FC = ({ children }) => {
+  const [selector, setSelector] = useState<WalletSelector | null>(null);
+  const [modal, setModal] = useState<WalletSelectorModal | null>(null);
+  const [accounts, setAccounts] = useState<Array<AccountState>>([]);
+
+  const init = useCallback(async () => {
+    const _selector = await setupWalletSelector({
+      network: "testnet",
+      debug: true,
+      modules: [
+        setupNearWallet(),
+        setupMyNearWallet(),
+        setupSender(),
+        setupMathWallet(),
+        setupNightly(),
+        setupLedger(),
+        setupWalletConnect({
+          projectId: "c8cb6204543639c31aef44ea4837a554",
```

Comment:
We should replace this with a placeholder

Reply:
Yeah forgot to change it after I tested if it was working correctly.